src/components/HomePageContent/ClusterInfo/ClusterInfo.module.css

```diff
@@ -0,0 +1,122 @@
+.clusterInfo {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  justify-content: center;
+  padding-bottom: 40px;
+}
+
+.clusterInfoBLockOne {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.clusterInfoText {
+  display: flex;
+}
+
+.mainButton {
+  width: 260px;
+  height: 48px;
+  border-radius: 8px;
+  padding: 10px 32px;
+  box-shadow: var(--shadow-primary);
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: var(--primary);
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 1.5;
+  color: var(--primary-dark);
+  transition: all var(--transition);
+}
```

Comment:
Do you really need to write this styles from the start, or just add/override styles to already existing Button component?